identity/validator_set_allegation.go

```diff
@@ -214,6 +220,7 @@ func (vs *ValidatorStore) ExecuteAllegationTracker(ctx *ValidatorContext, active
 			} else {
 				logger.Infof("Nothing to withdraw from addr on bounty program: %s\n", bountyAddress.Humanize())
 			}
+			processedValidators[ar.MaliciousAddress.Humanize()] = true
```

Comment:
This still does not solve the problem ,as the processedValidators ,list is created everytime . 
I added a script scripts/evidence.allegation_loadtest.go  , You can use  it to verify .

I think a possible fix would be 
- When you are setting the allegation , you can iterate over existing allegations and see if there is already an allegation existing against the validator.
-  When you freeze the validator, the alllegation must be deleted .

Just a suggestion ,I would be open to any other implementations you think might be better

Reply:
If a validator is frozen, we could not create an allegation requests (until defrost process)
If we have a several allegations on the one malicious, it will took into account the first from the requests list, other requests will be removed for this address (will push this part).